token/tokencontext.go

```diff
@@ -0,0 +1,23 @@
+package token
+
+import (
+	"context"
+
+	"github.com/fabric8-services/fabric8-wit/log"
+	"github.com/fabric8-services/fabric8-wit/login/tokencontext"
+	errs "github.com/pkg/errors"
+)
+
+// ReadManagerFromContext extracts the token manager from the context
+func ReadManagerFromContext(ctx context.Context) (*Manager, error) {
```

Comment:
maybe just add this function to token.go? It's a bit weird to see token/tokencontext.go when we have token/tokencontext package :)
I'm not sure why Hector didn't put such a function to tokencontext package (there is one which returns interface{}) but I guess because of cycle dependencies.

Reply:
yeah because of cyclic deps.
moved https://github.com/fabric8-services/fabric8-wit/pull/1637/commits/6acf5c877199d828616eea1e0395b16a40fdd86a